index.js

```diff
@@ -252,7 +262,7 @@ HtmlWebpackPlugin.prototype.executeTemplate = function (templateFunction, chunks
     .then(function () {
       var templateParams = {
         compilation: compilation,
-        webpack: compilation.getStats().toJson(),
+        webpack: getStats(compilation),
```

Comment:
I **think** this is equivalent to just passing `compilation`

I believe, the only difference between `compilation.getStats().toJson()` and `compilation` is that `compilation` have webpack types while `compilation.getStats().toJson()` gets us js native types

Not 100% sure on this one thought